### PR TITLE
[CI:DOCS] Remove 'you' from man pages

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

#### Does this PR introduce a user-facing change?

```release-note
The word you has been removed from all man pages
```

### CODE REVIEW

**Comment by rhatdan** [comment]
@TomSweeneyRedHat PTAL

**Comment by rhatdan** [comment]
@edsantiago PTAL

**Comment by edsantiago** [comment]
On my list for today, but please allow me some time, this is a big one.

File: docs/source/markdown/options/cpus.container.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 #### **--cpus**=*number*
 
 Number of CPUs. The default is *0.0* which means no limit. This is shorthand
-for **--cpu-period** and **--cpu-quota**, so you may only set either
+for **--cpu-period** and **--cpu-quota**, so specify either
```
Comment (edsantiago) [inline docs/source/markdown/options/cpus.container.md:8]: Not your fault (not your wording), but this sentence is hard to parse. Would you mind cleaning it up a little? 
>This is shorthand for [...] and cannot be set together
>This is shorthand for [...] and cannot be specified on the same command line
>This is shorthand for [...]. It is an error to combine **--cpus** with either of those.

File: docs/source/markdown/options/entrypoint.md [hunk 2]
```diff
@@ -6,16 +6,15 @@
 
 Overwrite the default ENTRYPOINT of the image.
 
-This option allows you to overwrite the default entrypoint of the image.
+This option overwrites the default entrypoint of the image.
```
Comment (edsantiago) [inline docs/source/markdown/options/entrypoint.md:9]: Actually, this should be **override**, not write (here and line 7).

File: docs/source/markdown/options/follow.md [hunk 3]
```diff
@@ -6,6 +6,6 @@
 
 Follow log output.  Default is false.
 
-Note: If you are following a <<container|pod>> which is removed by `podman <<container|pod>> rm`
-or removed on exit (`podman run --rm ...`), then there is a chance that the log
+Note: Following a <<container|pod>> which is removed by `podman <<container|pod>> rm`
```
Comment (edsantiago) [inline docs/source/markdown/options/follow.md:9]: (minor) I think '**WHEN** following ...' might be a little easier to understand.

File: docs/source/markdown/options/latest.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+####> This option file is used in:
+####>   podman attach, container inspect, diff, exec, init, inspect, kill, logs, mount, network reload, pause, pod inspect, pod kill, pod logs, pod rm, pod start, pod stats, pod stop, pod top, port, restart, rm, start, stats, stop, top, unmount, unpause, wait
+####> If you edit this file, make sure your changes
```
Comment (edsantiago) [inline docs/source/markdown/options/latest.md:3]: New use of "you" -- my fault, sorrynotsorry

**Review by edsantiago — COMMENTED** [review]
Review still in progress, but I get nervous about spending too long on github review: github has a nasty habit of clobbering comments when someone re-pushes. Please don't re-push yet, I'm still working on this.

File: docs/source/markdown/podman-container-diff.1.md [hunk 5]
```diff
@@ -1,54 +0,0 @@
-% podman-container-diff 1
```
Comment (Luap99) [inline docs/source/markdown/podman-container-diff.1.md:1]: This should not be removed. `podman diff` is not the same. It can accept either an image or container as argument while `podman container diff` only support containers.

File: docs/source/markdown/podman-generate-systemd.1.md [hunk 6]
```diff
@@ -237,7 +237,7 @@ WantedBy=default.target
 
 Podman-generated unit files include an `[Install]` section, which carries installation information for the unit. It is used by the enable and disable commands of systemctl(1) during installation.
 
-Once you have generated the systemd unit file, you can copy the generated systemd file to ```/etc/systemd/system``` for installing as a root user and to ```$HOME/.config/systemd/user``` for installing it as a non-root user. Enable the copied unit file or files using `systemctl enable`.
+Once the systemd unit file is generated, copy it to ```/etc/systemd/system``` for installing as a root user and to ```$HOME/.config/systemd/user``` for installing it as a non-root user. Enable the copied unit file or files using `systemctl enable`.
```
Comment (edsantiago) [inline docs/source/markdown/podman-generate-systemd.1.md:240]: Fault is in the original wording, but `and` should be **or** ("...or to ... for non-root").

File: docs/source/markdown/podman-mount.1.md.in [hunk 7]
```diff
@@ -12,12 +12,11 @@ podman\-mount - Mount a working container's root filesystem
 Mounts the specified containers' root file system in a location which can be
 accessed from the host, and returns its location.
 
-If you execute the command without any arguments, Podman will list all of the
-currently mounted containers, including external containers. External containers are
-containers in container/storage by tools other than Podman. For example Buildah and
-CRI-O.
+Podman list all of the currently mounted containers, including external containers, when the command is executed
```
Comment (edsantiago) [inline docs/source/markdown/podman-mount.1.md.in:15]: **lists**, with **s**, not list. The standalone "For example" sentence bothers me a little, but it was there in the original and I'm getting tired so I'll pretend I don't see it.

File: docs/source/markdown/podman-pod-create.1.md.in [hunk 8]
```diff
@@ -21,10 +21,10 @@ Name (“jonah”)
...
+for it. This name is useful to identify a pod.
 
 Note: resource limit related flags work by setting the limits explicitly in the pod's cgroup
-which by default, is the cgroup parent for all containers joining the pod. Containers are still delegated the ability to set their own resource limits when joining a pod meaning that if you run **podman pod create --cpus=5** you can also run **podman container create --pod=`<pod_id|pod_name>` --cpus=4** and the container will only see the smaller limit. containers do NOT get the pod level cgroup resources if they specify their own cgroup when joining a pod such as **--cgroupns=host**
+which by default, is the cgroup parent for all containers joining the pod. Containers are still delegated the ability to set their own resource limits when joining a pod meaning that if the pod was created via **podman pod create --cpus=5** specifying **podman container create --pod=`<pod_id|pod_name>` --cpus=4** causes the container see the smaller limit. containers do NOT get the pod level cgroup resources if their specified with their own cgroup when joining a pod such as **--cgroupns=host**
```
Comment (edsantiago) [inline docs/source/markdown/podman-pod-create.1.md.in:27]: This one makes my head hurt but I can't find a way to rephrase it. So, just two minor notes:
* comma wanted here: "when joining a pod >>,<< meaning ..."
* "Containers" needs to be capitalized in the last sentence
* **they're**, not their
* I can't count to two any more

File: docs/source/markdown/podman-pod-top.1.md.in [hunk 9]
```diff
@@ -7,17 +7,15 @@ podman\-pod\-top - Display the running processes of containers in a pod
 **podman pod top** [*options*] *pod* [*format-descriptors*]
 
 ## DESCRIPTION
-Display the running processes of containers in a pod. The *format-descriptors* are ps (1) compatible AIX format descriptors but extended to print additional information, such as the seccomp mode or the effective capabilities of a given process. The descriptors can either be passed as separated arguments or as a single comma-separated argument. Note that you can specify options and/or additionally options of ps(1); in this case, Podman will fallback to executing ps with the specified arguments and options in the container.
+Display the running processes of containers in a pod. The *format-descriptors* are ps (1) compatible AIX format descriptors but extended to print additional information, such as the seccomp mode or the effective capabilities of a given process. The descriptors can either be passed as separated arguments or as a single comma-separated argument. Note that if additionally options of ps(1) are specified, Podman falls back to executing ps with the specified arguments and options in the container.
```
Comment (edsantiago) [inline docs/source/markdown/podman-pod-top.1.md.in:10]: Not-your-fault but please remove the "d" from "separated". Should be "...as separate arguments"

And, "additionally" should be **additional**

File: docs/source/markdown/podman-port.1.md.in [hunk 10]
```diff
@@ -15,13 +15,9 @@ List port mappings for the *container* or look up the public-facing port that is
 
 #### **--all**, **-a**
 
-List all known port mappings for running containers.  When using this option, you cannot pass any container names
-or private ports/protocols as filters.
+List all known port mappings for running containers.  When using this option container names or private ports/protocols as filters can not be used.
```
Comment (edsantiago) [inline docs/source/markdown/podman-port.1.md.in:18]: Dropped the comma, please re-add: "when using this option COMMA container names ..."

Also, **cannot* is oneword.

File: docs/source/markdown/podman-run.1.md.in [hunk 11]
```diff
@@ -692,8 +691,8 @@ $ podman run --security-opt unmask=/foo/bar:/sys/firmware fedora bash
 
 ### Setting device weight
 
-If you want to set _/dev/sda_ device weight to **200**, you can specify the device
-weight by **--blkio-weight-device** flag. Use the following command:
+Specify the device weight by **--blkio-weight-device** flag.
+Use the following command:
```
Comment (edsantiago) [inline docs/source/markdown/podman-run.1.md.in:695]: The original "sda" and "200" text is lost, so instead of "Use the following command", it might be better to just say "Example:"

File: docs/source/markdown/podman-stats.1.md.in [hunk 12]
```diff
@@ -62,16 +62,13 @@ Valid placeholders for the Go template are listed below:
 
 [1] Cgroups V1 only
 
-When using a GO template, you may precede the format with `table` to print headers.
+When using a GO template, precede the format with `table` to print headers.
```
Comment (edsantiago) [inline docs/source/markdown/podman-stats.1.md.in:65]: Not-your-fault again, but the ALL-CAPS GO looks weird to my eye. Is that a standard convention, or should it be just *Go*? (Also present in a few other places)

File: docs/source/markdown/podman.1.md [hunk 13]
```diff
@@ -148,7 +148,7 @@ specify additional options via the `--storage-opt` flag.
 
 #### **--storage-opt**=*value*
 
-Storage driver option, Default storage driver options are configured in `containers-storage.conf(5)`. The `STORAGE_OPTS` environment variable overrides the default. The --storage-opt specified options overrides all. If you specify --storage-opt="", no storage options will be used.
+Storage driver option, Default storage driver options are configured in `containers-storage.conf(5)`. The `STORAGE_OPTS` environment variable overrides the default. The --storage-opt specified options overrides all. The --storage-opt option overrides all. Specify --storage-opt="" so no storage options will be used.
```
Comment (edsantiago) [inline docs/source/markdown/podman.1.md:151]: Duplicate duplication (the "overrides all" sentence is repeated).

**Comment by edsantiago** [comment]
P.S. I'm done for the time being. If anyone else is in the middle of a review, please speak up now. Otherwise, feel free to re-push.

File: docs/source/markdown/podman-container-diff.1.md (hunk 5 above)
Comment (rhatdan) [reply docs/source/markdown/podman-container-diff.1.md:1]: Readded.

File: docs/source/markdown/options/cpus.container.md (hunk 1 above)
Comment (TomSweeneyRedHat) [reply docs/source/markdown/options/cpus.container.md:8]: I'm not real clear what you're suggesting here @edsantiago, but I like the middle one of these three.

File: docs/source/markdown/options/entrypoint.md [hunk 14]
```diff
@@ -6,16 +6,15 @@
...
 
 The ENTRYPOINT of an image is similar to a COMMAND
 because it specifies what executable to run when the container starts, but it is
 (purposely) more difficult to override. The ENTRYPOINT gives a container its
-default nature or behavior, so that when you set an ENTRYPOINT you can run the
-container as if it were that binary, complete with default options, and you can
-pass in more options via the COMMAND. But, sometimes an operator may want to run
-something else inside the container, so you can override the default ENTRYPOINT
-at runtime by using a **--entrypoint** and a string to specify the new
+default nature or behavior. When the ENTRYPOINT is set the
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/options/entrypoint.md:14]:
```suggestion
default nature or behavior. When the ENTRYPOINT is set, the
```

File: docs/source/markdown/options/entrypoint.md [hunk 15]
```diff
@@ -6,16 +6,15 @@
...
+container runs as if it were that binary, complete with default options. More options can be
+passed in via the COMMAND. But, if a user wants to run
+something else inside the container, the **--entrypoint** option allows them to specify a new
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/options/entrypoint.md:17]:
```suggestion
something else inside the container, the **--entrypoint** option allows a new
```

File: docs/source/markdown/options/entrypoint.md [hunk 16]
```diff
@@ -6,16 +6,15 @@
...
 ENTRYPOINT.
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/options/entrypoint.md:18]:
```suggestion
ENTRYPOINT to be specified.
```

File: docs/source/markdown/options/latest.md (hunk 4 above)
Comment (TomSweeneyRedHat) [reply docs/source/markdown/options/latest.md:3]: The lines starting with `####>` are all auto-generated now, correct?

File: docs/source/markdown/options/network.md [hunk 17]
```diff
@@ -75,5 +75,4 @@ Valid _mode_ values are:
         host, using the loopback interface instead of the tap interface for improved
         performance
 
-    NOTE: For backward compatibility reasons, if you have a network named `pasta`,
-    Podman will use this instead of the pasta mode.
+    NOTE: For backward compatibility reasons, Podman will use networks named `pasta` instead of the pasta mode.
```
[the remark this answers is not in the document]
Comment (TomSweeneyRedHat) [reply docs/source/markdown/options/network.md:78]:
```suggestion
    NOTE: For backward compatibility reasons, if a network named `pasta` exists on the container, Podman will use it instead of the pasta mode.
```

File: docs/source/markdown/options/latest.md (hunk 4 above)
Comment (edsantiago) [reply docs/source/markdown/options/latest.md:3]: Yes, and not user-visible.

File: docs/source/markdown/options/pod.run.md [hunk 18]
```diff
@@ -4,6 +4,6 @@
 ####> are applicable to all of those.
 #### **--pod**=*name*
 
-Run container in an existing pod. If you want Podman to make the pod for you, prefix the pod name with **new:**.
+Run container in an existing pod. Podman can make the pod automatically by prefixing the pod name with **new:**.
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/options/pod.run.md:7]:
```suggestion
Run the container in an existing pod. Podman will make the pod automatically if the pod name is prefixed with **new:**.
```

File: docs/source/markdown/options/restart.md [hunk 19]
```diff
@@ -14,6 +14,5 @@ Valid _policy_ values are:
 - `always`                   : Restart containers when they exit, regardless of status, retrying indefinitely
 - `unless-stopped`           : Identical to **always**
 
-Please note that restart will not restart containers after a system reboot.
-If this functionality is required in your environment, you can invoke Podman from a **systemd.unit**(5) file, or create an init script for whichever init system is in use.
-To generate systemd unit files, please see **podman generate systemd**.
+Podman provides a systemd unit file podman-restart.service which restarts containers after a system reboot.
+Create a **systemd.unit**(5) file, to manage the lifecycle of a container as a system service. To generate systemd unit files, please see **podman generate systemd**.
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/options/restart.md:18]: Perhaps?
```suggestion
If this functionality is required in the local environment,  invoke Podman from a **systemd.unit**(5) file, or create an init script for whichever init system is in use.
```

File: docs/source/markdown/options/systemd.md [hunk 20]
```diff
@@ -8,7 +8,7 @@ Run container in systemd mode. The default is **true**.
 
 The value *always* enforces the systemd mode is enforced without
 looking at the executable name. Otherwise, if set to true and the
-command you are running inside the container is **systemd**, **/usr/sbin/init**,
+command running inside the container is **systemd**, **/usr/sbin/init**,
```
[the remark this answers is not in the document]
Comment (TomSweeneyRedHat) [reply docs/source/markdown/options/systemd.md:11]: The you removal looks fine here, but I concur with @edsantiago.  I'm not sure what this should be saying.

File: docs/source/markdown/options/restart.md (hunk 19 above)
Comment (rhatdan) [reply docs/source/markdown/options/restart.md]: Rewritten please check it out.

File: docs/source/markdown/options/add-host.md [hunk 21]
```diff
@@ -1,6 +1,6 @@
 ####> This option file is used in:
 ####>   podman build, create, pod create, run
-####> If you edit this file, make sure your changes
+####> If file is edited, make sure the changes
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/options/add-host.md:3]:
```suggestion
####> If this file is edited, make sure the changes
```

or perhaps `the`

Comment (edsantiago) [reply docs/source/markdown/options/add-host.md:3]: Oh please no. If it's absolutely necessary to un-you this part, *please* make it a separate PR. This PR is already unwieldy enough.

File: docs/source/markdown/options/restart.md [hunk 22]
```diff
@@ -14,6 +14,6 @@ Valid _policy_ values are:
 - `always`                   : Restart containers when they exit, regardless of status, retrying indefinitely
 - `unless-stopped`           : Identical to **always**
 
-Please note that restart will not restart containers after a system reboot.
-If this functionality is required in your environment, you can invoke Podman from a **systemd.unit**(5) file, or create an init script for whichever init system is in use.
-To generate systemd unit files, please see **podman generate systemd**.
+Podman provides a systemd unit file podman-restart.service which restarts containers after a system reboot.
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/options/restart.md:17]:
```suggestion
Podman provides a systemd unit file, podman-restart.service, which restarts containers after a system reboot.
```

File: docs/source/markdown/podman-pod-create.1.md.in [hunk 23]
```diff
@@ -21,10 +21,11 @@ Name (“jonah”)
 
 podman generates a UUID for each pod, and if a name is not assigned
 to the container with **--name** then a random string name will be generated
-for it. The name is useful any place you need to identify a pod.
+for it. This name is useful to identify a pod.
 
-Note: resource limit related flags work by setting the limits explicitly in the pod's cgroup
-which by default, is the cgroup parent for all containers joining the pod. Containers are still delegated the ability to set their own resource limits when joining a pod meaning that if you run **podman pod create --cpus=5** you can also run **podman container create --pod=`<pod_id|pod_name>` --cpus=4** and the container will only see the smaller limit. containers do NOT get the pod level cgroup resources if they specify their own cgroup when joining a pod such as **--cgroupns=host**
+Note: resource limit related flags work by setting the limits explicitly in the pod's cgroup parent 
+for all containers joining the pod. A container can be override the resource limits when joining a pod.
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-pod-create.1.md.in:27]:
```suggestion
for all containers joining the pod. A container can override the resource limits when joining a pod.
```

File: docs/source/markdown/podman-pod-create.1.md.in [hunk 24]
```diff
@@ -21,10 +21,11 @@ Name (“jonah”)
...
+For example, if a pod was created via **podman pod create --cpus=5**, specifying **podman container create --pod=`<pod_id|pod_name>` --cpus=4** causes the container to use the smaller limit. Also, containers whihc specify their own cgroup, such as **--cgroupns=host**, do NOT get the assigned the pod level cgroup resources.
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-pod-create.1.md.in:28]:
```suggestion
For example, if a pod was created via **podman pod create --cpus=5**, specifying **podman container create --pod=`<pod_id|pod_name>` --cpus=4** causes the container to use the smaller limit. Also, containers which specify their own cgroup, such as **--cgroupns=host**, do NOT get the assigned pod level cgroup resources.
```

File: docs/source/markdown/podman-port.1.md.in [hunk 25]
```diff
@@ -15,13 +15,9 @@ List port mappings for the *container* or look up the public-facing port that is
 
 #### **--all**, **-a**
 
-List all known port mappings for running containers.  When using this option, you cannot pass any container names
-or private ports/protocols as filters.
+List all known port mappings for running containers, when using this option container names or private ports/protocols filters cannot be used.
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-port.1.md.in:18]:
```suggestion
List all known port mappings for running containers; when using this option, container names or private ports/protocols filters cannot be used.
```

File: docs/source/markdown/podman-run.1.md.in [hunk 26]
```diff
@@ -137,11 +137,11 @@ and specified with a _tag_.
...
 
-At any time you can run **podman ps** in
-the other shell to view a list of the running containers. You can reattach to a
-detached container with **podman attach**.
+At any time run **podman ps** in
+the other shell to view a list of the running containers. Reattach to a
+detached container with **podman attach** command.
 
-When attached in the tty mode, you can detach from the container (and leave it
+When attached in the tty mode, detach from the container (and leave it
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-run.1.md.in:144]: Planted firmly on the fence on this one, but maybe?
```suggestion
When attached via tty mode, detach from the container (and leave it
```

File: docs/source/markdown/podman-run.1.md.in [hunk 27]
```diff
@@ -313,8 +313,8 @@ This is used to override the Podman provided user setup in favor of entrypoint c
 
 **Note:** If a container will be run within a pod, it is not necessary to publish the port for
 the containers in the pod. The port must only be published by the pod itself. Pod network
-stacks act like the network stack on the host - you have a variety of containers in the pod,
-and programs in the container, all sharing a single interface and IP address, and
+stacks act like the network stack on the host - meaning a variety of containers in the pod,
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-run.1.md.in:316]:
```suggestion
stacks act like the network stack on the host - meaning a variety of containers in the pod 
```

File: docs/source/markdown/podman-run.1.md.in [hunk 28]
```diff
@@ -465,14 +465,14 @@ $ podman run --read-only --read-only-tmpfs=false --tmpfs /run -i -t fedora /bin/
 
 ### Exposing log messages from the container to the host's log
 
-If you want messages that are logged in your container to show up in the host's
-syslog/journal then you should bind mount the _/dev/log_ directory as follows.
+Bind mount the _/dev/log_ directory to have messages that are logged in the container to show up in the host's
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-run.1.md.in:468]:
```suggestion
Bind mount the _/dev/log_ directory to have messages that are logged in the container  show up in the host's
```

File: docs/source/markdown/podman-run.1.md.in [hunk 29]
```diff
@@ -652,16 +651,16 @@ To disable the security labeling for this container versus running with the
 $ podman run --security-opt label=disable -i -t fedora bash
 ```
 
-If you want a tighter security policy on the processes within a container,
-you can specify an alternate type for the container. You could run a container
+Tighten security policy on the processes within a container by specifying an
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-run.1.md.in:654]: Soft suggestion
```suggestion
Tighten the security policy on the processes within a container by specifying an
```

File: docs/source/markdown/podman-start.1.md.in [hunk 30]
```diff
@@ -9,10 +9,10 @@ podman\-start - Start one or more containers
 **podman container start** [*options*] *container* ...
 
 ## DESCRIPTION
-Start one or more containers.  You may use container IDs or names as input.  The *attach* and *interactive*
+Start one or more containers using container IDs or names as input.  The *attach* and *interactive*
 options cannot be used to override the *--tty* and *--interactive* options from when the container
-was created. If you attempt to start a running container with the *--attach* option, podman will simply
-attach to the container.
+was created. Starting an already running container with the *--attach* option, podman simply
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-start.1.md.in:14]:
```suggestion
was created. Starting an already running container with the *--attach* option, Podman simply
```

File: docs/source/markdown/podman-stop.1.md.in [hunk 31]
```diff
@@ -9,8 +9,8 @@ podman\-stop - Stop one or more running containers
 **podman container stop** [*options*] *container* ...
 
 ## DESCRIPTION
-Stops one or more containers.  You may use container IDs or names as input. The **--time** switch
-allows you to specify the number of seconds to wait before forcibly stopping the container after the stop command
+Stops one or more containers using container IDs or names as input. The **--time** switch
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-stop.1.md.in:12]:
```suggestion
Stops one or more containers using container IDs or names as input. The **--time** option
```

File: docs/source/markdown/podman-volume-mount.1.md [hunk 32]
```diff
@@ -10,7 +10,7 @@ podman\-volume\-mount - Mount a volume filesystem
 Mounts the specified volumes' file system in a location which can be
 accessed from the host, and returns its location.
 
-Rootless mode only supports mounting file volumes, unless you enter the user namespace
+Rootless mode only supports mounting file volumes, unless Podman is run within the user namespace
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-volume-mount.1.md:13]:
```suggestion
Rootless mode only supports mounting file volumes unless Podman is run within the user namespace
```

File: docs/source/markdown/podman.1.md [hunk 33]
```diff
@@ -64,7 +64,7 @@ If `--hooks-dir` is unset for root callers, Podman and libpod will currently def
 
 Podman and libpod currently support an additional `precreate` state which is called before the runtime's `create` operation.  Unlike the other stages, which receive the container state on their standard input, `precreate` hooks receive the proposed runtime configuration on their standard input.  They may alter that configuration as they see fit, and write the altered form to their standard output.
 
-**WARNING**: the `precreate` hook lets you do powerful things, such as adding additional mounts to the runtime configuration.  That power also makes it easy to break things.  Before reporting libpod errors, try running your container with `precreate` hooks disabled to see if the problem is due to one of your hooks.
+**WARNING**: the `precreate` hook allows powerful things, such as adding additional mounts to the runtime configuration.  That power also makes it easy to break things.  Before reporting libpod errors, try running a container with `precreate` hooks disabled to see if the problem is due to one of the hooks.
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman.1.md:67]: soft suggestion
```suggestion
**WARNING**: the `precreate` hook allows powerful changes to occur, such as adding additional mounts to the runtime configuration.  That power also makes it easy to break things.  Before reporting libpod errors, try running a container with `precreate` hooks disabled to see if the problem is due to one of the hooks.
```

File: docs/source/markdown/podman.1.md [hunk 34]
```diff
@@ -148,7 +148,7 @@ specify additional options via the `--storage-opt` flag.
 
 #### **--storage-opt**=*value*
 
-Storage driver option, Default storage driver options are configured in `containers-storage.conf(5)`. The `STORAGE_OPTS` environment variable overrides the default. The --storage-opt specified options overrides all. If you specify --storage-opt="", no storage options will be used.
+Storage driver option, Default storage driver options are configured in `containers-storage.conf(5)`. The `STORAGE_OPTS` environment variable overrides the default. The --storage-opt specified options overrides all. Specify --storage-opt="" so no storage options will be used.
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman.1.md:151]:
```suggestion
Specify a storage driver option. Default storage driver options are configured in `containers-storage.conf(5)`. The `STORAGE_OPTS` environment variable overrides the default. The --storage-opt specified options override all. Specify --storage-opt="" so no storage options will be used.
```

**Comment by TomSweeneyRedHat** [comment]
I'm done.  I'm sure I missed some formatting (italic, bold, etc) issue, but the eyeballs are too tired to chase those at the moment.  LGTM overall and TY!
